src/api/player_view_functions.clj

```diff
@@ -1,5 +1,6 @@
 (ns api.player-view-functions
   (:require [rules.victory-conditions :as victory]
+            [rules.abilities :as abilities]
             [configs.messages :as messages]))
 
 (defn ^:private partial-card
```

Comment:
I feel like "filter-card" or something would be a more appropriate name considering what it does... It filters only the selected keys and reassigns the owner (which maybe should be done separately?